src/stave.ts

```diff
@@ -328,7 +319,7 @@ export class Stave extends Element {
   }
 
   // Text functions
-  setText(
+  setStaveText(
```

Comment:
Was this renamed because it conflicts with the new Element.text() method? Makes sense.

Reply:
Yes, this is right.

Reply:
Can things like this be documented in the `docs/changelog/5.0.md` file? 

This breaking change might trip up someone trying to upgrade from VexFlow 4.x to 5.0. If they have code that calls setText(...) for some reason, it will now call the incorrect setText(). Instead, their code should call `setStaveText(...)`.